proto/osmosis/lockup/query.proto

```diff
@@ -33,6 +35,8 @@ service Query {
 
 	// Returns account locked records with longer duration
 	rpc AccountLockedLongerThanDuration(AccountLockedLongerDurationRequest) returns (AccountLockedLongerDurationResponse);
+	// Returns account locked records with longer duration excluding tokens started unlocking
+	rpc AccountLockedLongerDurationNotUnlockingOnly(AccountLockedLongerDurationNotUnlockingOnlyRequest) returns (AccountLockedLongerDurationNotUnlockingOnlyResponse);
```

Comment:
Let's either add in word `Than` to this function name, or remove it from the above one

Reply:
I would like to keep names shorter, removed Than from above function

Reply:
Now the keeper functions have the word "Than" it them (i.e. https://github.com/osmosis-labs/osmosis/pull/41/files#diff-eaa0d5418b72ff86c277ac9083dc98ac3cd54dcea1aaca7a51a134dd4133da21R131), but the rpc functions do not.  Should we unify these?

Reply:
Fixed it